src/MoBi.Presentation/Presenter/MoBiMainViewPresenter.cs

```diff
@@ -21,15 +22,18 @@ public interface IMoBiMainViewPresenter : IMainViewPresenter,
       IListener<SimulationRunStartedEvent>,
       IListener<SimulationRunFinishedEvent>,
       IListener<ReportCreationStartedEvent>,
-      IListener<ReportCreationFinishedEvent>
+      IListener<ReportCreationFinishedEvent>,
+      IListener<ProjectLoadedEvent>,
+      IListener<ProjectClosedEvent>
```

Comment:
What about ProjectSaved event? If I save the project under a different name, the title bar should be updated as well

Reply:
Good point, I'll check if that behavior is missing now, and changed if necessary

Reply:
The behavior with 'save as' indeed was wrong. Fixed it.